### PR TITLE
Add url output

### PR DESCRIPTION
**Pull request by bollwyvl**
Thanks for this action!

This adds an `output` which other steps can use. Perhaps I don't understand how it all works properly, but it seems like this might enable #10 through other actions, and let this action stay fairly simple.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -37,6 +37,7 @@ async function run() {
     core.debug('Linking to:')
     core.debug(url)
     core.debug((new Date()).toTimeString())
+    core.setOutput("url", url);
```
Comment (bollwyvl) [inline index.js:40]: hooray one-liner!

**Comment by choldgraf** [comment]
btw @larsoner - I'm getting so much use out of this action :-)  are there other ways that we could help to improve it?

**Comment by larsoner** [comment]
Thanks @bollwyvl ! Let me know if you need me to make a new release for this or something.

> I'm getting so much use out of this action :-) are there other ways that we could help to improve it?

If people are bored, #7 needs fixing and someday it would be nice to have unit tests that actually test outputs. And (more fun) maybe now that this PR is in, someone could figure out how to combine the output with some other action to make #10 work!

**Comment by bollwyvl** [comment]
Thank you so much!

Tagged releases would be very helpful... @master gives me the willies.

As to testing... yeah, a first step might be to do the light port to
typescript, as it can catch a bunch of stuff at transpile time, as most
(all) of the upstreams are already typed, and it wouldn't add much to the
build or runtime footprint. The circle payload would be mostly untyped,
unfortunately. I guess a sufficient test would be with some fixtures of
good and bad json.

>

**Comment by larsoner** [comment]
> Tagged releases would be very helpful... @master gives me the willies.

https://github.com/larsoner/circleci-artifacts-redirector-action/releases/tag/0.2.0

> As to testing...

My JavaScript knowledge is very limited, so I'm happy to review and merge any such fixes but at least to start I'm probably not the best person to implement them...

**Comment by bollwyvl** [comment]
Happy to oblige, might be a few days before I get the chance! JavaScript is
indeed a terrible systems language, but here we are!

**Pull request merged**